src/App.tsx

```diff
@@ -3,8 +3,45 @@ import './App.css';
 import { Layout } from "antd";
 import { Menu } from "antd";
 import "antd/dist/antd.css";
+import { Table } from "antd"
```

Comment:
Ini dijadiin satu aja sama import "antd" diatas

```suggestion
import { Menu, Table } from "antd"
```